packages/create/index.js

```diff
@@ -95,76 +95,30 @@ const PLUGINS = {
       },
       format: value => path.resolve(value),
     },
-    {
-      type: 'select',
-      name: 'language',
-      message: 'Language',
-      choices: [
-        {
-          title: 'TypeScript (Recommended)',
-          value: 'ts',
-        },
-        {
-          title: 'JavaScript',
-          value: 'js',
-        },
-      ],
-    },
-    {
-      type: 'multiselect',
-      name: 'plugins',
-      message: 'How would you like to render your animation?',
-      choices: [
-        {
-          title: 'Image sequence',
-          value: 'core',
-          disabled: true,
-          selected: true,
-        },
-        {
-          title: 'Video (FFmpeg)',
-          value: 'ffmpeg',
-          selected: true,
-        },
-      ],
-      warn: 'This option is always included.',
-      onRender() {
-        this.value[0].selected = true;
-      },
-      validate: value => {
-        if (!Array.isArray(value)) {
-          return '"plugins" option must be an array.';
-        }
-
-        if (!value.includes('core')) {
-          return 'Image sequence is required.';
-        }
-
-        return true;
-      },
-    },
   ]);
 
-  if (!response.plugins) {
+  if (!response.path) {
     console.log(kleur.red('× Scaffolding aborted by the user.\n'));
     return;
   }
 
+  const plugins = [PLUGINS.core, PLUGINS.ffmpeg];
+  const language = 'ts';
```

Comment:
I think we can just hardcode the ts in `template-2d-ts` and `vite.config.ts`.

Reply:
done